RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -31,36 +31,38 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
         self.items = items
     }
 
-    func deleteItem(item: Item) {
+    func delete(item: Item) {
         viewController.uiWrite {
-            guard !(item as Object).invalidated, let index = items.indexOf(item) else {
+            guard !item.isInvalidated, let index = items.index(of: item) else {
                 return
             }
 
             items.realm?.delete(item)
-            viewController.tableView.deleteRowsAtIndexPaths([NSIndexPath(forRow: index, inSection: 0)], withRowAnimation: .Left)
+            viewController.tableView.deleteRows(at: [IndexPath(row: index, section: 0)], with: .left)
+            viewController.didUpdateList(reload: false)
```

Comment:
Okay. I went through the commit history with a fine-toothed comb to see what happened.

You originally removed `didUpdateList` when you added the `uiWrite` blocks to both the `delete` and `complete` methods since that call is now made internally at the end of that block. When I merged in the latest changes from the `master` branch, they got added back in.

Since the update calls are no longer necessary in either method, I removed them.